doc/ref_impl/cntt-ri/chapters/chapter06.md

```diff
@@ -5,15 +5,25 @@
 
 ## Table of Contents
 * [6.1 Introduction](#6.1)
-* [6.2 Heading](#6.2)
-* [6.3 Heading](#6.3)
+* [6.2 OPNFV Pharos Specification](#6.2)
+* [6.3 Spine Leaf Topology](#6.3)
 
 <a name="6.1"></a>
 ## 6.1 Introduction
 
 <a name="6.2"></a>
-## 6.2 Heading
+## 6.2 OPNFV Pharos Specification
+
+The Reference Implemenation will follow the baseline OPNFV Pharos specification.  The following diagam illustrates the networking
+and servers required.
+
+<img src="../figures/ch06_lab_tpology.jpg" title="Lab Topology">
 
 
 <a name="6.3"></a>
-## 6.3 Heading
+## 6.3 Spine Leaf Topology
+
+The ideal spine leaf topology would span at least 3 racks to ensure HA at the rack level as demonstrated in the follow diagram.
```

Comment:
@markbeierl Hi Mark, Generally, the RI is prescriptive, and further, the initial RI is primarily intended to support RC (i.e. a 5 node config isn't going into prod). There are a lot of details around increasing the availability of the platform and other best practices, and I think we all find ourselves getting on that track from time to time. However, I don't think we want to start trying to describe the ideal design for a production environment, within the scope of this RI, at this time, so I'd vote that we ditch the 3 racks, yada, yada. Similarly, it's probably best not to try to prescribe what will be L2 and what will be L3 in the fabric, at this time. If for no other reason, the fabric in the current RI is pure L2. We can discuss further, if you'd like. Thanks, -Mark

Reply:
@markbeierl , I agree.  The requirements for RI and RC likely won't span multiple racks, etc.  I think it add confusion when try to align the actual physical setup (i.e. 6 nodes, or so) with comments about 3 racks and many switches.

Reply:
Spine/leaf multi rack references removed.